### DISCUSSION TITLE
Getting error CsvHelper.MissingFieldException: 'Field at index '82' does not exist.

### DISCUSSION THREAD

**Discussion by sumitshekhar41** [JoshClose/CsvHelper#2298] [Q&A]
When I am processing less records(2) in csv, its processing, while I am increasing number of records, I am getting following error

CsvHelper.MissingFieldException: 'Field at index '82' does not exist. You can ignore missing fields by setting MissingFieldFound to null.
IReader state:
   ColumnCount: 0
   CurrentIndex: 82
   HeaderRecord:

IParser state:
   ByteCount: 0
   CharCount: 4096
   Row: 4
   RawRow: 4
   Count: 82
   RawRecord:

**Comment by AltruCoder** [comment]
You would get this error if you were missing a field in your row that CsvHelper was expecting to be there.  In the following example, in the 4th row it is missing a comma after "Bill", so there is no `LastName` field in that row.

```csharp
void Main()
{
	var data = @"Id,FirstName,LastName
	1,Tom,Jones
	2,Jenny,Adams
	3,Bill
	";
	
	using var reader = new StringReader(data);
	using var csv = new CsvReader(reader, CultureInfo.InvariantCulture);
	
	var records = csv.GetRecords<Foo>().ToList();
}

public class Foo
{
	public int Id { get; set; }
	public string FirstName { get; set; }
	public string LastName { get; set; }
}

```
